iina/Base.lproj/PrefGeneralViewController.xib

```diff
@@ -525,22 +524,57 @@
                         <binding destination="IBP-Mz-Maa" name="value" keyPath="values.playlistShowMetadataInMusicMode" id="nxW-hX-9co"/>
                     </connections>
                 </button>
+                <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="AbF-Dw-gXa">
+                    <rect key="frame" x="118" y="7" width="260" height="18"/>
+                    <buttonCell key="cell" type="check" title="Enable repeat automatically upon start" bezelStyle="regularSquare" imagePosition="left" state="on" inset="2" id="j4o-RP-pfA">
```

Comment:
Missing a colon after "start". I looked over the settings and that is a consistent convention, except for one other setting, `Check for updates` is also missing a colon at the end of the label.